src/redux/books/books.js

```diff
@@ -0,0 +1,27 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+const initialState = [];
+
+export const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
+export const removeBook = (payload) => ({
+  type: REMOVE_BOOK,
+  payload,
+});
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.payload];
+    case REMOVE_BOOK:
+    //   console.log(state);
```

Comment:
### [OPTIONAL]

- [ ] It will be great to remove this commented code in your project. This will keep your work clean and professional. Remember, you can always use git history to move back to the previous version of your code.

Reply:
Done, Thank you.

Reply:
This issue is well resolved! ✔️